src/io/buffered.cr

```diff
@@ -1,12 +1,10 @@
-# The `IO::Buffered` mixin enhances the `IO` module with input/output buffering.
+# The `IO::Buffered` mixin enhances an `IO` with input/output buffering.
```

Comment:
Why`IO::Buffered` should be a module and `IO::Delimited` a class?

Reply:
We could make `IO::Buffered` a class too, but then you'd have to do like in Java:

```crystal
io = File.open("some_file")
io = IO::Buffered.new(io)
io.gets
```

otherwise things will be very inefficient. That's why in Ruby buffering is embedded in these kind of IOs because it's almost always slow otherwise.

We could have `IO::Buffering` as a module and `IO::Buffered` as a class, though, or something like that.

Still, maybe a discussion for another issue :-)